meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,323 @@
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "sortFacetValuesBy": {"*": "count"}
+  }'
+```
+
+Alternatively, it is possible to order a single facet by `count`, while other attributes follow alphanumeric ordering:
+
+```bash
+ curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings/faceting \
+  -H 'Content-Type: application/json'  \
+--data-binary '{
+    "sortFacetValuesBy": {"*": "alpha", "genre": "count"}
+  }'
+```
+
+Done by @Kerollmops in (#3612)
+
+## Language support improvements
+
+* Enhance Japanese word segmentation (#218) @mosuka
+* Enhance separator-based Tokenization (#215) @ManyTheFish
+  - words containing `_` are now properly segmented into several words
+  - brackets `{([])}` are no longer considered as context separators for the `proximity` ranking rule
+
+Done by @ManyTheFish, @mosuka in [#3866] and in [Charabia v0.8.1](https://github.com/meilisearch/charabia/releases/tag/v0.8.1)
+
+
+## Display a total count of tasks on the `/tasks/` route
+The `/tasks` route now displays the total number of tasks in the task queue with a `total` property. It also displays the total number of tasks within a specific filter e.g. you can view the total number of successfully processed tasks with `/tasks?statuses=succeeded`. This detail gives insight into the progress of processed tasks over time.
+
+Done by @Kerollmops in (#3889)
+
+## Other improvements
+
+
+* Improve the Prometheus `/metrics` experimental feature ([#625](https://github.com/meilisearch/product/discussions/625)). Provides metrics on the task queue such as the number of queued tasks and the number of processing tasks. Adds the real database size used by Meilisearch. Add "meilisearch" prefix to metrics (#3789, #3861) @irevoire @dureuill 
+* Reduce index size (~15%) by removing an internal database (#3819) @loiclec
+* Add new `/experimental-features` endpoint to configure `scoreDetails` and `vectorStore`  (#3850) @dureuill 
+* Reduce deserialization time by using `RoaringBitmap::deserialize_unchecked_from` (#3788) @Kerollmops 
+* Re-enable autobatching for addition and deletion tasks (#3670) @irevoire 
+* Expose lastUpdate and isIndexing in /stats endpoint (#3851) @gentcys
+
+
+# Fixes 🐞
+* Fix matching results returned by the `exactness` ranking rules when there are hard separators present (#3824) @dureuill 
+* Fixes an issue where searching for words without `.` doesn't match any string in the index ([#151](https://github.com/meilisearch/product/discussions/151), [#3778](https://github.com/meilisearch/meilisearch/discussions/3778))
+* Fix an issue around incorrect deletion of documents when using the `disableOnAttributes` typo setting  (#3819) @loiclec
+* Replace the `atty` dependency with the `is-terminal` due to security vulnerability (#3878) @Kerollmops
+* Fix analytics properties name total_received on multiple events (#3877) @dureuill 
+* Adds matchingWords and maxMatchingWords score details (#3885) @dureuill 
```

Comment:
This PR is a fix following the merge of the "display score detail" feature. You can remove this line from the final changelog since the users need only to know the feature is released, no matter was the "path" to reach this.
If you want the PR to be mentioned anyway, you can add it in the related section (not mandatory, really if you want)

```
Done by @dureuill in #3771 and #3885
```